usagov/vote-gov-drupal
# VOTE-2993 nvrf data directory update

Update NVRF to v0.18.2



## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2993

## Description

Update NVRF to v0.18.2 and change location of nvrf data directory

## Deployment and testing

### Post-deploy steps

1. cd into vote_nvrf and run `npm i`
2. run `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/es and confirm you can access the spanish homepage.
2. visit http://vote-gov.lndo.site/es/registrar/idaho/llenador-de-formulario-por-correo-postal and test the NVRF in Spanish
3. visit http://vote-gov.lndo.site/register/idaho/mail-in-form-filler and test the NVRF in English.

## Checklist for the Developer




## Checklist for the Peer Reviewers
